components/brave_federated_learning/DEPS

```diff
@@ -0,0 +1,10 @@
+include_rules = [
+  "+base",
```

Comment:
AFAIK base and url are considered reduntand, because they are used everywhere, but np to keep them

Reply:
👍